src/components/ToDoList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ToDoItem from './ToDoItem.js'
+import './ToDoList.css'
+import items from '../items.js'
+
+class ToDoList extends React.Component {
+	constructor() {
```

Comment:
1. Do not forget about passing constructor arguments to super (parent) class. Example:
constructor(props) {
super(props)
		this.state = {
			todos: items
		}
	}

Reply:
Do I need to do a constructor in all of my class components?

Reply:
No, define it only if you have some logic for constructor. But if you define it - do not forget about "super(props)"